scheduler/cloudformation/cloudformation.go

```diff
@@ -274,9 +277,138 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		}
 	}
 
+	go s.waitForDeploymentToStabilize(ctx, app, stackName, events, done)
 	return nil
 }
 
+func (s *Scheduler) waitForDeploymentToStabilize(ctx context.Context, app *scheduler.App, name string, events scheduler.EventChan, done chan error) {
+	if events == nil {
+		events = make(scheduler.EventChan)
+	}
+
+	events <- scheduler.Event{Message: fmt.Sprintf("Updating stack \"%s\"", name)}
+	if err := <-done; err != nil {
+		events <- scheduler.Event{Error: err}
+		return
+	}
+	events <- scheduler.Event{Message: "Stack updated"}
```

Comment:
Since we don't return an error if the lock is canceled this outputs "Stack updated" if it hasn't actually updated but has just been superseded